### ISSUE TITLE
Add Yubikey Support for serivces that support it

### ISSUE THREAD

**Issue by 333fred** [meetfranz/franz#1362]
I use a yubikey for 2fa on Google/Facebook/anywhere else that supports it, and for Google I have a required policy that I must use my hardware key to log in. Franz doesn't support yubikeys, so I can't use them as 2fa sources, meaning I can't log into my gmail/gcalendar at all and have to use an alternate 2fa source for Facebook. I would love it if Franz supported yubikeys so I could use these services with it.

**Comment by 333fred** [comment]
Well, please don't. It's still a problem.

**Comment by RakuDigital** [comment]
Yeah, +1 to this. Trying to sustain Franz in my Enterprise tech stack is relatively difficult without Yubikey (or other usb key devices) support. Totally fine if it's a paid feature, but please consider implementing this.

**Comment by nikunjkotecha** [comment]
+1 to this, please add support of Yubikey
